templates/core/features/ota-updates.html

```diff
@@ -1,218 +1,201 @@
-      <p>The kernel and base operating system are handled as snaps as well, so all the benefits that apply to applications also apply to the system foundation. This means not only benefitting from fast and consistent updates to the core, but also graceful error handling with automatic rollbacks on improperly updated kernels.</p>
-    </div>
-    <div class="col-5 u-hide--medium u-hide--small u-align--center">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/714e4c31-OTA_Stack_Case.svg",
-        alt="Safe operating system updates",
-        width="300",
-        height="130",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
-    </div>
-  </div>
-</section>
-
-<section class="p-strip--light">
-  <div class="row u-equal-height">
-    <div class="col-5 u-hide--medium u-hide--small u-align--center">
-          {{ image (
-            url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
-            alt="Get in touch",
-            width="281",
-            height="200",
-            hi_def=True,
-            loading="lazy"
-            ) | safe
-          }}
-    </div>
-    <div class="col-7">
-      <h2>Get an IoT app store for OTA updates</h2>
-      <p>Get your own infrastructure for secure management and seamless software updates for your fleet of devices.</p>
-      <p>
-        <a href="/core/contact-us?product=core-ota-updates" class="p-button--positive js-invoke-modal">Get in touch</a>
-      </p>
-    </div>
-  </div>
-</section>
+  <section class="p-section--hero">
+    <div class="u-fixed-width p-section--shallow">
+      <h1 class="u-no-margin--bottom">Over-the-air updates for Linux, done right</h1>
+      <h2>Ubuntu Core sets a new standard for Linux device updates, covering  the kernel, the OS and applications.</h2>
+    </div>
+    <div class="row--50-50">
+      <div class="col">
+        <p>
+          Transactional updates for reliability. Deltas minimize network traffic. Digital signatures guarantee integrity and provenance.
+        </p>
+        <p class="p-section--shallow">This is the future of smart things.</p>
+        <hr class="is-muted" />
+        <a class="p-button--positive" href="/internet-of-things/appstore">Get an IoT app store</a>
+        <a href="/engage/snapd-whitepaper">OTA updates whitepaper&nbsp;&rsaquo;</a>
+      </div>
+      <div class="col u-hide--medium u-hide--small">
+        <div class="p-image-container--3-2 is-highlighted">
+          <img class="p-image-container__image"
+               src="https://assets.ubuntu.com/v1/46f1b890-IoT_field_updates_from_Cloud.svg"
+               alt="Iot field updates from cloud"
+               width="550" />
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width p-section--shallow">
+      <hr />
+      <h2>Reliability</h2>
+    </div>
+    <div class="row--25-75-on-large">
+      <div class="col">
+        <div class="p-equal-height-row">
```

Comment:
Looks like the first gif needs replacing as the top space doesn't match the other two
![Screenshot from 2024-08-30 09-22-38](https://github.com/user-attachments/assets/b3be6b55-a6df-415e-b98f-8641c2f1014b)

Reply:
@mattea-turic please check this out too